index.html

```diff
@@ -5,310 +5,243 @@
+                    </li>
+                    <li class="lang">
+                        <img src="pics/works img/Icons/html icon.svg" alt="">
+                        <span>HTML</span>
+                    </li>
+                    <li class="lang">
+                        <img src="pics/works img/Icons/css icon.svg" alt="">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <li class="languages border">
+                    <span class="Lang-title">Framework</span>
+                    <i class="fa fa-angle-right" style="font-size: 24px"></i>
+                </li>
+                <li class="languages border">
+                    <span class="Lang-title">Skills</span>
+                    <i class="fa fa-angle-right" style="font-size: 24px"></i>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="form-sec">
+        <div class="bg">
+            <div class="form">
+                <div class="form-title">
+                    <h3>Contact me</h3>
+                    <p class="form-desc">If you have an application you are interested in developing, a feature that you
+                        need built or a project
+                        that needs coding. I’d love to help with it </p>
                 </div>
-                <form id="input-contact-box" action="https://formspree.io/f/xpzbqlab" method="post">
-                    <label class="label" for="username"></label>
-                    <input id="username" class="input-i" name="username" type="text" maxlength="30"
-                        placeholder="Enobong" required />
-                    <label class="label" for="email"></label>
-                    <input id="email" class="input-i" name="email" type="email" value="" placeholder="misong4real@yahoo.com"
-                        required />
-                    <label class="label" id="input-message" for="msg"></label>
-                    <textarea class="input-m" name="msg" id="msg" maxlength="500" cols="20" rows="10"
-                        placeholder="Write your message here" required></textarea>
-                    <button class=" get-touch-btn" type="submit">Get In Touch</button>               
-                   </form>
-                  </div>
-            </section>
-      </main>
-   </body>
-</html>
+                <form action="https://formspree.io/f/mgebkndp"  method="POST" class="mainForm">
+                    <input class="input" type="text" maxlength="30" id="name" required placeholder="Enobong">
+                    <input class="input" type="email" id="mail" required placeholder="misong4real@yahoo.com">
+                    <textarea id="msg" name="user-msg" maxlength="500" required placeholder="Write your message here"></textarea>
+                    <button class="form-btn" type="submit">Get in touch</button>
+                </form>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please fix the margin & padding of the form and the container holding the form so that the form content can take up more space on the small screen. This was highlighted by previous reviewers. 👍🏾 
![form](https://user-images.githubusercontent.com/99656687/221350230-530a9add-0ba7-42ed-9261-91a29fc51b29.png)